### ISSUE TITLE
Get Element States keyword in presenter mode

### ISSUE THREAD

**Issue by tomeekCZ** [MarketSquare/robotframework-browser#2068]
**Describe the bug**
When in presenter mode, the keyword **Get Element States** does not return state **detached** for element, which is detached. It will raise timeout error instead.

**To Reproduce**
```
*** Settings ***
Library     Browser    enable_presenter_mode=False


*** Test Cases ***
Test
    New Browser    headless=False
    New Page
    Go To    https://robotframework.org/
    ${state_visible_element}    Get Element States    //h1[@class='title']    evaluate    visible in value
    ${state_detached_element}    Get Element States    //h1[@class='wrong_title']    evaluate    detached in value
```

**Expected behavior**
When presenter mode is not active, the code works just fine. When I set presenter mode as active, the first Get Element State works (the element is there), but the second one doesn't. The element is not there, but it should return detached instead of the error message.

**Desktop (please complete the following information):**
 - OS: Windows 10, build 19044.1706
 - Browser: Chromium 
 - Version: 102.0.5005.40 (Developer Build) (64-bit)

**Comment by jannerau** [comment]
Could it be that this is not expected behavior after all?

I've got keyword which uses Get Element States to get boolen value for hidden state (quite similar as example in Browser library keyword documentation) and it works just fine in presenter mode with Browser library v.12.1.0 but does not work with any later library versions.

**Comment by Snooz82** [comment]
Will be fixed with #2109
